modules/ROOT/pages/sandboxes.adoc

```diff
@@ -52,3 +52,4 @@ To test and install each upgrade:
 == See Also
 
 * xref:update-acm.adoc[Upgrade API Community Manager]
+* https://videos.mulesoft.com/watch/RjG1iwDhtGNMgyLKmUrPDC[Creating Community Sandboxes]
```

Comment:
Looks good! I recommend updating the link so it's marked as an external link to a page outside of docs.mulesoft.com, and opens in a new tab. The links in the "See Also" section at the bottom of the index page of the ACM docs are like this. For example:

```
https://help.mulesoft.com/s/knowledge[Knowledge Base Articles, role=external, window=blank]
```

Reply:
Done